### PR TITLE
chore: upgrade ui kit to webpack 5

### PR DESCRIPTION
**Pull request by clintonmedbery**
# Description

This upgrades UI Kit to Webpack 5. 

## Which issue(s) does this PR relate to?
 - https://jira.d2iq.com/browse/D2IQ-96026 

## Testing

Really just click around and make sure it works. Bonus points for using `npm link` or copying the dist to kommander ui and testing things.

## Trade-offs

Should be none.

## Checklist

- [x] This PR is associated with a JIRA and it is mentioned in the commit message footer ("Closes …")
- [x] Significant changes have been tested downstream to avoid breaking changes
- [ ] This PR contains breaking changes and states in the commit message body ("BREAKING CHANGE: …")
- [x] I have reviewed the changes and provided detail to the sections above

### CODE REVIEW

File: webpack/webpack.base.js [hunk 1]
```diff
@@ -19,14 +19,13 @@ const webpackBase = {
       },
       {
         test: /\.stories\.tsx?$/,
-        loaders: [
```
Comment (clintonmedbery) [inline webpack/webpack.base.js:22]: Loaders is gone in webpack 5

**Review by seialkali — APPROVED** [review]
Looks good! Tested this in ui-kit and kommander-ui, and didn't see any funny business. Great work 💥

File: .github/workflows/storybook-tests.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Install Playwright 🎭
         run: npx playwright install --with-deps
       - name: Run Storybook Tests 📚
-        run: npm run build:storybook --quiet
```
Comment (mstruebing) [inline .github/workflows/storybook-tests.yml:19]: Why do we remove  this flag?

Comment (clintonmedbery) [reply .github/workflows/storybook-tests.yml:19]: Not really trying to remove it. The tests are not running properly and I am trying to debug it 😩

**Review by nataliepina — APPROVED** [review]
Awesome! Thank you for the effort put into making all these changes to unblock us.

**Pull request merged**